learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md

```diff
@@ -102,25 +102,30 @@ Follow the steps below to start a remote debug session.
 
 2. Open the Terminal and execute the Ballerina command, which you want to debug, out of the supported remote debugging commands below. 
 
-    - Debugging a Ballerina package/ single file: 
+    - Debugging a Ballerina package or a single file: 
 
     ```bash
     bal run --debug <DEBUGGEE_PORT> <BAL_FILE_PATH/PACKAGE_PATH>
     ```
 
-    - Debugging ballerina tests: 
+   - Debugging Ballerina executable jar:  
```

Comment:
```suggestion
   - Debugging Ballerina executable:  
```

Reply:
Executable jar looks better to me, as using just "executable" might leave some doubt (executable code, an executable file, or an executable program?)

Reply:
@praneesha WDYT?

Reply:
+1 say "executable JAR". Note the uppercase letters in the word "JAR".

Reply:
Updated in f8d2254